charts/internal/shoot-system-components/charts/csi-driver-manila/templates/daemonset-node.yaml

```diff
@@ -0,0 +1,229 @@
+          securityContext:
+            privileged: true
+            capabilities:
+              add: ["SYS_ADMIN"]
+            allowPrivilegeEscalation: true
+          image: {{ index $.Values.images "csi-driver-manila" }}
+          args:
+            - /bin/manila-csi-plugin
+            - --nodeid=$(NODE_ID)
+            {{- if $.Values.csimanila.runtimeConfig.enabled }}
+            - --runtime-config-file=/runtimeconfig/runtimeconfig.json
+            {{- end }}
+            {{- if $.Values.csimanila.topologyAwarenessEnabled }}
+            - --with-topology
+            - --nodeaz={{ $.Values.csimanila.nodeAZ }}
+            {{- end }}
+            - --endpoint=/csi/csi.sock
+            - --drivername=$(DRIVER_NAME)
+            - --share-protocol-selector=$(MANILA_SHARE_PROTO)
+            - --fwdendpoint=/csi-fwd/{{ .fwdNodePluginEndpoint.sockFile }}
+            - --cluster-id={{ $.Values.csimanila.clusterID }}
+            - --v=2
+          env:
+            - name: DRIVER_NAME
+              value: {{ .protocolSelector | lower }}.manila.csi.openstack.org
+            - name: NODE_ID
+              valueFrom:
+                fieldRef:
+                  fieldPath: spec.nodeName
+            - name: MANILA_SHARE_PROTO
+              value: "{{ .protocolSelector }}"
+          ports:
+            - containerPort: {{ .node.livenessProbe.healthPort }}
+              name: healthz
+              protocol: TCP
+          livenessProbe:
+            failureThreshold: 5
+            httpGet:
+              path: /healthz
+              port: healthz
+            initialDelaySeconds: 30
+            timeoutSeconds: 10
+            periodSeconds: 30
+          volumeMounts:
+            - name: {{ .protocolSelector | lower }}-plugin-dir
+              mountPath: /csi
+            - name: {{ .protocolSelector | lower }}-fwd-plugin-dir
+              mountPath: /csi-fwd
+            {{- if $.Values.csimanila.runtimeConfig.enabled }}
+            - name: {{ .protocolSelector | lower }}-runtime-config-dir
+              mountPath: /runtimeconfig
+              readOnly: true
+            {{- end }}
+            {{- if $.Values.openstack.caCert }}
+            - name: manila-csi-plugin
+              mountPath: /var/run/csi-manila
+              readOnly: true
+            {{- end }}
+
+{{- if $.Values.resources.driverNode }}
+          resources:
+{{ toYaml $.Values.resources.driverNode | indent 12 }}
+{{- end }}
+
+        - name: driver-nfs-{{ .protocolSelector | lower }}-node
+          securityContext:
+            privileged: true
+            capabilities:
+              add: ["SYS_ADMIN"]
+            allowPrivilegeEscalation: true
+          image: {{ index $.Values.images "csi-driver-nfs" }}
+          args :
+            - "--v=2"
+            - "--nodeid=$(NODE_ID)"
+            - "--endpoint=$(CSI_ENDPOINT)"
+            - "--drivername=gardener-{{ .protocolSelector | lower }}.nfs.csi.k8s.io"
```

Comment:
`gardener-{{ .protocolSelector | lower }}.nfs.csi.k8s.io` there a `.nfs` which you may have missed.